bower.json

```diff
@@ -39,9 +38,9 @@
     "*.md"
   ],
   "resolutions": {
-    "d3": "^3.3.13"
+    "d3": "~3.3.13"
   },
   "devDependencies": {
-    "d3": "^3.3.13"
+    "d3": "~3.3.13"
```

Comment:
@liquidpele, can you enumerate those changes that require 3.5.x, so that we can add unit tests for them?